packages/contracts-ops/contracts/CallBatch.sol

```diff
@@ -0,0 +1,117 @@
+// SPDX-License-Identifier: MIT OR Apache-2.0
+pragma solidity 0.7.6;
+
+// solhint-disable quotes
```

Comment:
?

Reply:
json demands `"` but the linter also wants `"` for strings, which would require lots of escapes `"\""` so we use `'` in solidity, and silence the linter